streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -689,7 +703,6 @@ public void close(boolean clean,
         try {
             suspend(clean, isZombie);
         } catch (final RuntimeException e) {
-            clean = false;
```

Comment:
Why do we remove this? If we don't need it in `closeSuspended()` we should update `closeSuspended()` to not take `clean` as parameter

Reply:
oh, mistake!